velox/exec/NestedLoopJoinProbe.cpp

```diff
@@ -401,6 +401,9 @@ RowVectorPtr NestedLoopJoinProbe::doMatch(vector_size_t probeCnt) {
   VELOX_CHECK(!hasProbedAllBuildData());
 
   if (joinCondition_ == nullptr) {
+    probeMatched_.setAll();
```

Comment:
Please add a comment here about why these need to be updated (You can use some of the text from the PR description)